app/views/dashboards/_analytics.html.erb

```diff
@@ -1,4 +1,4 @@
-<% num_views = @articles.sum(&:page_views_count) %>
+<% _num_views = @articles.sum(&:page_views_count) %>
```

Comment:
The underscores before the variable names can be removed now since we disabled the "unused variable assignment" cop for the views. Such variable names are used for the unused variables, so your fix is technically correct. But actually, the variables are used. So the cop seems to not work correctly and I've disabled it for now.
(You may need to `pull` from master to get the new `erblint` config)

Reply:
However I have also done this fix in other places and they have already been merged, I hope this doesn't cause any issues.

Reply:
I believe that won't cause any issues, just wanted to clarify this.